src/pages/SubmissionConfig.js

```diff
@@ -125,19 +128,26 @@ export const SubmissionConfig = ({ methodId }) => {
         ])
       ]),
       div({ style: { marginTop: '2rem', height: '2rem', fontWeight: 'bold' } }, ['Select a data table']),
-      h(Select, {
-        isDisabled: false,
-        'aria-label': 'Select a data table',
-        isClearable: false,
-        value: selectedRecordType ? selectedRecordType : null,
-        onChange: ({ value }) => {
-          setSelectedRecordType(value)
-          loadRecordsData(value)
-        },
-        placeholder: 'None selected',
-        styles: { container: old => ({ ...old, display: 'inline-block', width: 200 }) },
-        options: _.map(t => t.name, recordTypes)
-      }),
+      div({}, [
+        h(Select, {
+          isDisabled: false,
+          'aria-label': 'Select a data table',
+          isClearable: false,
+          value: selectedRecordType ? selectedRecordType : null,
```

Comment:
So definitely a small complaint... but I wonder whether it would be possible to still show the name of the table here, with the warning sign next to it? 

I think it would be a nice hint to the user that (eg) `sample` is the table that the workflow is configured to use, but because it doesn't exist, we can't run the workflow yet. (but maybe if they imported it, we could run it?)

And I think it should still be safe to fill that value in... the user still can't submit the workflow, and it should still be obvious that something's not right.

Reply:
Maybe this is a dumb thought, but wouldn't that be confusing to the user? It seems contradicting to me to tell a user, "Hey we didn't find this data table" but then show that data table in the list of imported data tables, as opposed to omitting it from the list which may then signal the user that they need to import it since it's not in the list. Maybe I could add a pop-up on the icon to suggest importing the data table?

Reply:
👍 based on our conversation this morning, I think (a) it's a lot harder and (b) I was outvoted on usefulness, so feel free to ignore this comment for now (until we get more user feedback)!